recipe/run_test.sh

```diff
@@ -39,7 +39,7 @@ python -m numba.tests.test_runtests
 
 if [[ "$archstr" == 'aarch64' ]]; then
 	echo 'Running only a slice of tests'
-	$SEGVCATCH python -m numba.runtests -b -j --random='0.15' --exclude-tags='long_running' -m $TEST_NPROCS -- numba.tests
+	$SEGVCATCH python -m numba.runtests -b -j --random='0.10' --exclude-tags='long_running' -m $TEST_NPROCS -- numba.tests
```

Comment:
I reduced it from 15% to 10% of the tests, but it still times out.

Reply:
Unfortunately, I suspect if that 10% still contains the parfors test(s) it will still time out.